framework/src/functormaterials/ParsedFunctorMaterial.C

```diff
@@ -22,10 +22,11 @@ ParsedFunctorMaterialTempl<is_ad>::validParams()
       "Computes a functor material from a parsed expression of other functors.");
   params.addRequiredCustomTypeParam<std::string>(
       "expression", "FunctionExpression", "Expression to parse for the new functor material");
-  params.addParam<std::vector<std::string>>("functor_names",
-                                            "Functors to use in the parsed expression");
+  params.addParam<std::vector<std::string>>(
+      "functor_names", {}, "Functors to use in the parsed expression");
   params.addParam<std::vector<std::string>>(
```

Comment:
Do people use this without passing any functors? Just wondering whether we should make this required (without defaults) instead

Reply:
I dont know if they do but I would expect it to work.
If it did not I would switch to a GenericFunctorMaterial but that s because I know all the functor materials lol